### PR TITLE
add quota error to tsg

### PR DESCRIPTION
**Pull request by heoelri**

### CODE REVIEW

File: docs/reference-implementation/Troubleshooting.md [hunk 1]
```diff
@@ -50,6 +51,24 @@ Sorry, we are currently experiencing high demand in this region, and cannot fulf
...
+
+Often times followed by more details about the affected region, the current usage and the additional required quota:
+
+```console
+Location: SwedenCentral, Current Limit: 100, Current Usage: 96, Additional Required: 8, (Minimum) New Limit Required: 104. 
+```
+
+**Description:** Occurs when a deployment requires more cores than the current quota allows.
+
+**Solution:** Either reduce the number of cores used, or request more quota for a given VM SKU size in a given region. See [regional quota requests](https://docs.microsoft.com/azure/azure-supportability/regional-quota-requests) for more details.
```
Comment (sebader) [inline docs/reference-implementation/Troubleshooting.md:70]: ...or try a different region?

Comment (heoelri) [reply docs/reference-implementation/Troubleshooting.md:70]: That would of course also work.

Comment (heoelri) [reply docs/reference-implementation/Troubleshooting.md:70]:
```suggestion
**Solution:** Either reduce the number of cores used, request more quota for a given VM SKU size in a given region or switch to another region that provides the requested quota. See [regional quota requests](https://docs.microsoft.com/azure/azure-supportability/regional-quota-requests) for more details.
```

**Review by sebader — APPROVED** [review]

**Comment by sebader** [comment]
please port to -connected

**Pull request merged**